Modules/ThirdParty/DICOMParser/src/DICOMParser/DICOMParser.h

```diff
@@ -75,6 +75,11 @@ class DICOM_EXPORT DICOMParser
   //
   bool OpenFile(const dicom_stl::string& filename);
 
+  //
+	// Closes the currently opened file.
+	//
+	void CloseFile();
```

Comment:
```suggestion
  // Closes the currently opened file.
  void CloseFile();
```
These should be spaces, not tabs. We don't need empty comment lines, though for consistency that might need to be changed throughout this entire file.